api/v1alpha1/openstackclusterstackrelease_types.go

```diff
@@ -28,8 +28,25 @@ type OpenStackClusterStackReleaseSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of OpenStackClusterStackRelease. Edit openstackclusterstackrelease_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// The name of the cloud to use from the clouds secret
+	CloudName string `json:"cloudName"`
+
+	// IdentityRef is a reference to a identity to be used when reconciling this cluster
+	IdentityRef OpenStackIdentityReference `json:"identityRef,omitempty"`
+}
+
+// OpenStackIdentityReference is a reference to an infrastructure
+// provider identity to be used to provision cluster resources.
+type OpenStackIdentityReference struct {
```

Comment:
It is somehow possible to import this type from CAPO?

Reply:
I can check if it is possible

Reply:
Thank you

Reply:
Hey, I think you can import existing types, but let's do it after https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1797 to reduce dependencies.

Reply:
The mentioned PR is merged in CAPO v1alpha8

Reply:
IMO this PR could be closed on behalf of https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/pull/23. 

Once CAPO v1alpha8 is available https://pkg.go.dev/sigs.k8s.io/cluster-api-provider-openstack@v0.9.0/api then the `renovate` bot will do this job for us and reduce CAPO dependencies.